src/handlebars/sponsors.handlebars

```diff
@@ -18,6 +18,10 @@
     <p>The AdoptOpenJDK Foundation's Tier-1 infrastructure providers contribute the largest share of infrastructure to the Adopt OpenJDK build farm project. Without these companies, the project would not be able to provide the quality, speed and availability
       of test coverage that it does today. (Listed alphabetically).</p>
     <table class="sponsor-table" notranslate>
+      <tr>
+        <td><a target="_blank" href="https://www.azul.com/"><img src="dist/assets/azul.png" alt="Azul Logo"></a></td>
```

Comment:
azul.png needs to be added to src/assets I believe otherwise this is a broken image reference

Reply:
Azul is already in there (not sure how that happened but there you go)